### ISSUE TITLE
Error seen in Canvas (post edit page) after composer update (v 4.2.3)

### ISSUE THREAD

**Issue by connecteev** [cnvs/canvas#508]
Error seen in Canvas (post edit page) after composer update (v 4.2.3). Doing a php artisan view:clear did not help.

http://127.0.0.1:8001/canvas/posts/b8d80c34-e87e-4ee7-821b-d7379dcb78ea/edit

```
/Users/kunalpunjabi/Code/kb_skins/storage/framework/views/1e22d9a38f0e125eebc911814cefafe769580868.php
                    <div class="col-12">
                        <label for="twitter_title" class="font-weight-bold"><?php echo e(__('canvas::posts.forms.seo.twitter.title.label')); ?></label>
                        <input type="text" class="form-control border-0 px-0"
                               name="twitter_title" title="<?php echo e(__('canvas::posts.forms.seo.twitter.title.label')); ?>" value="<?php echo e($data['meta']['twitter_title']); ?>"
                               placeholder="<?php echo e(__('canvas::posts.forms.seo.twitter.title.placeholder')); ?>">
                    </div>
                </div>
                <div class="form-group row">
                    <div class="col-12">
                        <label for="twitter_description" class="font-weight-bold"><?php echo e(__('canvas::posts.forms.seo.twitter.description.label')); ?></label>
                        <textarea name="twitter_description" class="form-control border-0 px-0" rows="1"
                                  placeholder="<?php echo e(__('canvas::posts.forms.seo.twitter.description.placeholder')); ?>"
                                  title="<?php echo e(__('canvas::posts.forms.seo.twitter.description.label')); ?>"><?php echo e($data['meta']['twitter_description']); ?></textarea>
                    </div>
                </div>
                <div class="form-group row">
                    <div class="col-12">
                        <label for="canonical_link" class="font-weight-bold"><?php echo e(__('canvas::posts.forms.seo.canonical.label')); ?></label>
                        <input type="text" class="form-control border-0 px-0"
                               name="canonical_link" title="<?php echo e(__('canvas::posts.forms.seo.canonical.label')); ?>" value="<?php echo e($data['meta']['canonical_link']); ?>"
                               placeholder="<?php echo e(__('canvas::posts.forms.seo.canonical.placeholder')); ?>">
                    </div>
                </div>
            </div>
            <div class="modal-footer">
                <button class="btn btn-link text-muted" data-dismiss="modal"><?php echo e(__('canvas::buttons.general.done')); ?></button>
            </div>
        </div>
    </div>
</div>
<?php /**PATH /Users/kunalpunjabi/Code/kb_skins/vendor/cnvs/canvas/src/../resources/views/components/modals/post/edit/seo.blade.php ENDPATH**/ ?>
Arguments
"Undefined index: canonical_link (View: /Users/kunalpunjabi/Code/kb_skins/vendor/cnvs/canvas/resources/views/components/modals/post/edit/seo.blade.php) (View: /U ▶"
```

**Comment by austintoddj** [comment]
Thanks for the heads up @connecteev!

Fixed in [v4.2.4](https://github.com/cnvs/canvas/releases/tag/v4.2.4)

**Comment by connecteev** [comment]
@austintoddj thanks for the super quick fix! Confirmed that this is working in 4.2.4
